fixtures/envelope-with-profile.json

```diff
@@ -0,0 +1,5 @@
rofiler-test/dist/esbuild/index.js","module":"index"},{"lineno":28,"colno":30421,"function":"La","abs_path":"/Users/jonasbadalic/code/node-profiler-test/dist/esbuild/index.js","module":"index"},{"lineno":0,"colno":0,"in_app":false,"function":"(garbage collector)","abs_path":""},{"lineno":28,"colno":30421,"function":"La","abs_path":"/Users/jonasbadalic/code/node-profiler-test/dist/esbuild/index.js","module":"index"},{"lineno":28,"colno":30421,"function":"La","abs_path":"/Users/jonasbadalic/code/node-profiler-test/dist/esbuild/index.js","module":"index"},{"lineno":28,"colno":30421,"function":"La","abs_path":"/Users/jonasbadalic/code/node-profiler-test/dist/esbuild/index.js","module":"index"},{"lineno":28,"colno":30421,"function":"La","abs_path":"/Users/jonasbadalic/code/node-profiler-test/dist/esbuild/index.js","module":"index"},{"lineno":28,"colno":30421,"function":"La","abs_path":"/Users/jonasbadalic/code/node-profiler-test/dist/esbuild/index.js","module":"index"}],"stacks":[[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16],[17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[44,45,46,47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[45,46,47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[46,47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[48,49,44,45,46,47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[50,16],[49,44,45,46,47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[51,52,48,49,44,45,46,47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[52,48,49,44,45,46,47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16],[53,54,55,51,52,48,49,44,45,46,47,43,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,9,10,11,12,13,14,15,16]],"thread_metadata":{"0":{"name":"main"}}},"transaction":{"name":"fib: sourcemaps here","id":"66578634d48d433db0ad52882d1efe5b","trace_id":"33321bfbd5304bcc9663d1b53b08f84e","active_thread_id":"0"}}
```

Comment:
Not saying this is too large but if there's a "minimally viable profile file" that's smaller than this, I'd prefer that 😅